### ISSUE TITLE
Add version selection to advanced settings.

### ISSUE THREAD

**Issue by sbliven** [doomeer/factorio#71]
It would be nice to use the planner with older Factorio versions as well.

**Comment by doomeer** [comment]
I agree :) Ideally I would like to implement a feature where one can edit all the recipes (using some kind of JSON configuration file for instance, possibly automatically extracted from the game data). So one would be able to use older versions, but also mods.

**Comment by sbliven** [comment]
What's the current process for updating to new versions? Is there a script to dump the item recipes and generate recipes.ml?

I'd be interested in implementing this feature, but I don't want to break the update process.

**Comment by doomeer** [comment]
No script yet unfortunately. I, and contributors, edit the recipe.ml file by hand. If you do that, I will gladly merge the pull request. However, if you want to implement old recipes, or mod recipes, this will have to be a fork of the project.

By the way, if you want the old recipes, you can checkout an older version of Factorio Planner. Open index.html and you'll have the old version recipes :)
